includes/widgets/rating.php

```diff
@@ -66,6 +66,30 @@ private function add_style_tab() {
 			]
 		);
 
+		$this->add_responsive_control(
+			'icon_gap',
+			[
+				'label' => esc_html__( 'Spacing', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'range' => [
+					'em' => [
+						'min' => 0,
+						'max' => 10,
+						'step' => 0.1,
+					],
+					'rem' => [
+						'min' => 0,
+						'max' => 10,
+						'step' => 0.1,
+					],
+				],
+				'size_units' => [ 'px', 'em', 'rem', 'vw', 'custom' ],
+				'selectors' => [
+					'{{WRAPPER}}' => '--e-rating-gap: {{SIZE}}{{UNIT}}',
```

Comment:
I am not sure about that.
AFAIK, as the implementation of `--e-rating-gap` is at the moment we will have vertical and horizontal gaps.
Shouldn't the gap be horizontal only?